docs/source/conf.py

```diff
@@ -14,7 +14,18 @@
 
 import sys
 import os
-from pych.version import APP_NAME, APP_VERSION
+
+# copied from module/setup.py
+# This is hacky, but it means install_requires can be defined below.
+with open('../../module/pych/version.py', 'r') as fp:
```

Comment:
I suppose we should make the file path platform independent. Wrapping it in `os.path.abspath` should do it. Do you mind changing that here and in setup.py?